.github/workflows/test.yml

```diff
@@ -35,6 +35,9 @@ jobs:
           - odoo-version: 17.0
             python-version: "3.10"
             machine: ubuntu-22.04
+          - odoo-version: 18.0
+            python-version: "3.12"
```

Comment:
We'll test with python 3.10, since this is the minimum supported version, per
https://github.com/odoo/odoo/blob/9e35989436b807d5db738116cd3982dcb4efe81c/setup.py#L66

To be revisited/confirmed when Odoo release their official Docker image.

Reply:
Oh, but this is to test the template itself, so ok.